CMakeLists.txt

```diff
@@ -40,6 +40,11 @@ if (NOT CONFIG_MENDER_SERVER_TENANT_TOKEN)
 else()
     message(STATUS "Using custom '${CONFIG_MENDER_SERVER_TENANT_TOKEN}' server tenant token")
 endif()
+if (NOT CONFIG_MENDER_DEVICE_TYPE)
+    message(STATUS "Using default device type")
+else()
+    message(STATUS "Using custom '${CONFIG_MENDER_DEVICE_TYPE}' device type")
+endif()
```

Comment:
For yours and other readers information,
The main `CMakeLists.txt` file is ignored when building the project in Zephyr mode.

Reply:
Should I not touch this file?

Reply:
We can touch it as much as we want to make our POSIX build nice, but I am just saying that is ignored on the integration builds.